tests/processes/BearRunningTest.py

```diff
@@ -280,6 +279,7 @@ def test_run(self):
                              LOG_LEVEL.DEBUG,
                              LOG_LEVEL.DEBUG,
                              LOG_LEVEL.WARNING]
+
```

Comment:
sorry didn't want to confuse you. As you see you just add a newline in the "nowhere", meaning adding or removing such newlines is not necessary for your change. So please don't remove any existing empty lines or add them :)